src/Analyzer.Core.BuiltInRuleTests/TestRunner.cs

```diff
@@ -30,35 +30,38 @@ public static void Initialize(TestContext testContext)
         /// <param name="ruleExpectations">The test configuration to run.</param>
         [DataTestMethod]
         [DynamicData(nameof(GetTests), DynamicDataSourceType.Method, DynamicDataDisplayName = nameof(GetTestDisplayName))]
-        public void TestRule(TestConfiguration ruleExpectations)
+        public void TestRules(TestConfiguration ruleExpectations)
         {
+            // Verify test config is valid (decided in GetTests function below).
+            // Reason for failure is described in RuleId.
+            if (ruleExpectations.RuleId.StartsWith("Invalid test"))
```

Comment:
Was this mentioned in the Readme?

Reply:
"Invalid test" doesn't follow the requirements for built-in rule names, so I don't think it's worth mentioning.